matching-notification-ui/src/main/resources/MatchingNotification/MatchingRunInfoClass.xml

```diff
@@ -91,17 +91,53 @@
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.DateClass</classType>
     </startedTime>
-    <numPatientsUsedForLastMatchRun>
+    <numPatientsCheckedForMatches>
       <customDisplay/>
       <disabled>0</disabled>
-      <name>numPatientsUsedForLastMatchRun</name>
+      <name>numPatientsCheckedForMatches</name>
       <number>4</number>
-      <prettyName>Number of local patients updated</prettyName>
+      <prettyName>Number of local patients processed</prettyName>
       <size>8</size>
       <unmodifiable>0</unmodifiable>
       <validationMessage/>
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
-    </numPatientsUsedForLastMatchRun>
+    </numPatientsCheckedForMatches>
+    <numErrors>
```

Comment:
Don't manually edit these XMLs, this XClass is not valid since it's missing a `<numberType>` setting.

Reply:
I just re-used a piece of some other XML we have in PT, whats a `numberType`? This document can be imported using theimport option without errors, so whats not valid?